doc/dox_comments/header_files/ssl.h

```diff
@@ -4423,6 +4489,31 @@ WOLFSSL_API int wolfSSL_X509_NAME_get_text_by_NID(
 */
 WOLFSSL_API int wolfSSL_X509_get_signature_type(WOLFSSL_X509*);
 
+/*!
+    \brief This function frees an external WOLFSSL_X509 structure.
+
+
+    \param x509 a pointer to the WOLFSSL_X509 struct.
+
+    _Example_
+    \code
+    WOLFSSL_X509* x509 = (WOLFSSL_X509)XMALOC(sizeof(WOLFSSL_X509), NULL,
```

Comment:
The cast from XMALLOC should be to a pointer to WOLFSSL_X509, for example

WOLFSSL_X509* x509 = (WOLFSSL_X509*)...